contrib/devtools/security-check.py

```diff
@@ -5,6 +5,9 @@
 Otherwise the exit status will be 1 and it will log which executables failed which checks.
 Needs `readelf` (for ELF) and `objdump` (for PE).
 '''
+
+import sys; assert sys.version_info < (3,), ur"This script does not run under Python 3. Please use Python 2.7.x."
+
 from __future__ import division,print_function,unicode_literals
```

Comment:
From the CI:
  ```
File "../contrib/devtools/security-check.py", line 11
    from __future__ import division,print_function,unicode_literals
SyntaxError: from __future__ imports must occur at the beginning of the file
```

Reply:
Yep. This was expected, and why I didn't want to change stuff outside of rpc-test dir